os_migrate/plugins/module_utils/reference.py

```diff
@@ -249,6 +249,26 @@ def subnet_pool_ref(conn, id_, required=True):
     return _fetch_ref(conn, conn.network.find_subnet_pool, id_, required)
 
 
+def user_id(conn, ref, required=True):
```

Comment:
We probably don't need these methods yet but they look good so feel free to keep them.